team-norms/delivery_healthcheck.md

```diff
@@ -0,0 +1,17 @@
+# Delivery Healthcheck
+
+It is the responsibility of the lead on each delivery team to provide an update on the health of their delivery in each week's BizOps showcase.
+
+A Delivery Healthcheck should answer:
+
+## Current iteration goal
+ - When was your current iteration started?
+ - When is your next showcase?
+ - Are you on track to meet customer expectations?
+
+## Next iteration goal
+ - What is the goal for your next iteration?
+
+## Retrospective
+ - When was your last retrospective?
+ - What were your last retrospective action points?
```

Comment:
All the other questions are pretty punchy - this is one that could go long, so worth keeping an eye on.

If retros only come out with a couple of action points, likely not a problem. If they have 5+ then this could take a while!

Reply:
Yep - let's see how this one pans out over the first few weeks and adjust accordingly